GoogleUtilities/Network/GULNetworkURLSession.m

```diff
@@ -158,6 +158,12 @@ - (nullable NSString *)sessionIDFromAsyncPOSTRequest:(NSURLRequest *)request
                                             delegate:self
                                        delegateQueue:[NSOperationQueue mainQueue]];
     postRequestTask = [session uploadTaskWithRequest:request fromData:request.HTTPBody];
+
+    // Xcode 15 Beta 4: A URLRequest should not have a non-nil `HTTPBody`.
+    NSData *data = [request.HTTPBody copy];
+    NSMutableURLRequest *compliantRequest = [request mutableCopy];
+    compliantRequest.HTTPBody = nil;
+    postRequestTask = [session uploadTaskWithRequest:compliantRequest.copy fromData:data];
```

Comment:
Seems like there are two options, do this or add a new API that allows a NSData parameter.